File: 백준/Bronze/1546. 평균/평균.js
let fs = require("fs");

let input = fs.readFileSync("dev/stdin").toString().split("\n");

const N = Number(input[0]);

const scores = input[1].split(" ").map(Number);
const max = Math.max(...scores);

const newScores = scores.map(score => score/max * 100);
const sum = newScores.reduce((acc,cur)=> acc+cur, 0);
const avg = sum / N;
console.log(avg);

File: 백준/Bronze/1546. 평균/평균.cc
#include <iostream>

using namespace std;

int main(){
    int N;
    cin >> N;
    float a[1000];
    float max = 0;
    float sum=0;
    for(int i=0; i<N; i++){
        cin >> a[i];
        if(a[i]>max) max = a[i];
    }
    for(int i=0; i<N; i++){
        a[i] = a[i]/max*100;
        sum += a[i];
    }
    cout << sum/N;

}